src/client.js

```diff
@@ -156,6 +157,12 @@ module.exports = class Client {
     return this
   }
 
+  disableCase (value = true) {
+    this._disableCase = value
```

Comment:
And then follow suit here with:
```suggestion
  bodyCasing (value = true) {
    this._bodyCasing = value
```

The main reason for this is doing `.disableCase(false)` words weird compared to `.bodyCasing(false)` and `.bodyCasing(true)`